Assets/GothicVR/Scripts/Phoenix/Util/MusicColliderHandler.cs

```diff
@@ -0,0 +1,32 @@
+using UnityEngine;
+using GVR.Util;
+using GVR.Creator;
+using System.Collections.Generic;
+
+namespace GVR.Phoenix.Util
+{
+    public class MusicCollisionHandler : MonoBehaviour
+    {
+        private static List<string> musicZones = new List<string>();
+
+        void OnTriggerEnter(Collider other)
+        {
+            if (!other.CompareTag("Player")) return;
+
+            musicZones.Add(gameObject.name);
+
+            SingletonBehaviour<MusicCreator>.GetOrCreate().SetMusic(gameObject.name, MusicCreator.Tags.Std);
```

Comment:
Is it creating, loading, or just setting a value?
Can you be more precise, please. As the creators were for creation only before.

Reply:
Since we are using the Singleton pattern, we can use the object created by [PhoenixImporter.cs](https://github.com/GothicVRProject/GothicVR/blob/6ae529fffc7df0b16c5144f3e56f8628c57f2d9d/Assets/GothicVR/Scripts/Importer/PhoenixImporter.cs#L129) everywhere, since it's supposed to only be one instance, to mix and update the music.